packages/stratocacher-layer-rls/src/index.js

```diff
@@ -0,0 +1,30 @@
+import {getNamespace} from "request-local-storage";
+import {Layer} from "stratocacher";
+import Q from "q";
+
+function getCache() {
+	const rls = getNamespace();
```

Comment:
This is creating (and then discarding) a new namespace for each cache access. It will never have a cache hit. 😆 

We can probably just use a single namespace initialized at the module level. The `RLS` function that's returned from `getNamespace` can be called per cache access to retrieve the storage object for the current request. Also, we can just store directly in the `RLS()` object. It is not shared with any other RLS users. No need for a nested `stratocacher` object.

Reply:
Ahh right, makes sense. 👍